app/env.js

```diff
@@ -26,6 +26,8 @@ const vars = envalid.cleanEnv(
     AUTH_AUDIENCE: envalid.str({ devDefault: 'inteli-dev' }),
     AUTH_ISSUER: envalid.url({ devDefault: 'https://inteli.eu.auth0.com/' }),
     AUTH_TOKEN_URL: envalid.url({ devDefault: 'https://inteli.eu.auth0.com/oauth/token' }),
+    IDENTITY_SERVICE_HOST: envalid.host({ devDefault: 'localhost' }),
+    IDENTITY_SERVICE_PORT: envalid.port({ devDefault: 3001 }),
```

Comment:
Well the identity service is already in the `docker-compose.yaml` so if it would conflict badly it already will be. Having said that, yes, I'd like this merged first